utils/v2/sudo.ts

```diff
@@ -0,0 +1,28 @@
+import { api, Extrinsic, sudo } from "./setup";
+import { User } from "../User";
+import { signSendFinalized } from "./event";
+import { MangataGenericEvent } from "@mangata-finance/sdk";
+import { SudoDB } from "../SudoDB";
+
+export class Sudo {
```

Comment:
can you add all this to the existing sudo class instead?

Reply:
lets do a follow up patch to merge v2 into existing structure if all works ok... ?

Reply:
Once you finish with the fix of sigle.scenarios.tests I'll create another branch and move only productive code. I'm not happy with the idea of having WIP code in the main branch.